docs/src/tutorials/bank_fraud_detection_with_tfdf.ipynb

```diff
@@ -0,0 +1,1316 @@
+      "text/plain": [
+       "<Figure size 1024x450 with 3 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "tp.add_index(dataset_tp, \"CUSTOMER_ID\").plot(indexes=\"3774\")\n",
+    "\n",
+    "# Same plot as:\n",
+    "# tp.filter(dataset_tp, tp.equal(dataset_tp[\"CUSTOMER_ID\"], \"3774\")).plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b2f4a4fb-fc01-48a8-b69c-6fdf96ea65eb",
+   "metadata": {},
+   "source": [
+    "After exploring the dataset, we want to compute some augmented features that may correlate with fraudulent activities. We will compute the following three features:\n",
+    "\n",
+    "**Calendar features**: We will extract the hour of the day and the day of the week as individual features. This is because fraudulent transactions may be more likely to occur at specific times.\n",
```

Comment:
**Calendar features**: Extract the hour of the day and the day of the week as individual features. This is useful because fraudulent transactions may be more likely to occur at specific times.

Reply:
*Done